src/leaderboard.js

```diff
@@ -1,45 +1,61 @@
-function refreshScores(scoresContainer) {
+async function refreshScores(scoresContainer, gameId) {
```

Comment:
- [ ] For this activity, let's ensure arrow function is used for function. Our function can look like:

```javascript
...
const refreshScores= async (scoresContainer, gameId) => {
 ...
}
```